### DISCUSSION TITLE
为啥Umi4要抛弃umi-request

### DISCUSSION THREAD

**Discussion by calimanco** [umijs/umi#8859] [Q&A] [answered]
从Umi3升级到Umi4发现这块改动挺大的，之前很多处理是走中间件，现在只能改成拦截器，不免想吐槽。
之前吹umi-request比axios强多少，现在不就啪啪打脸了嘛

**Comment by fz6m** [comment, marked as answer]
不维护了。https://github.com/umijs/umi-next/issues/692
